README.md

```diff
@@ -34,102 +32,104 @@ _/bricks/lean-welcome.html_:
 
 
 ## How Does LEGO Work?
 
-LEGO (_Lightweight Embedded Gluten-free Objects_) is a base class that you extend and provide you with methods to use ♻️ reactive and 🏡 **native web-components** that are easy to digest 🌱 for your browser.
+LEGO (_Lightweight Embedded Gluten-free Objects_) is a base class that you can extend and provide you with methods to use ♻️ reactive and 🏡 **native Web Components** that are easy to digest 🌱 for your browser.
 
 A small CLI in NodeJS come along to build 🚀 your component.
-There's no [Vite](https://vitejs.dev/) nor other 3rd parties, it's [just an interpreter of ±180 lines of readable code](https://github.com/Polight/lego/blob/master/src/compiler/transpiler.js).
-I also works with [https://bun.sh](Bun) and probably other interpreters.
+There's no [Vite](https://vitejs.dev/) nor other 3rd parties, it's [just an interpreter of ±130 lines of readable code](https://github.com/Polight/lego/blob/master/src/compiler/transpiler.js).
+
+It also works with [Bun](https://bun.sh) and probably other interpreters.
 
 
 ## What Is LEGO Behind?
 
-- 👙 Minimalist: ~74~ 61 lines of readable code in its core (non-optimised, uncompressed, no cheating)
-- 🌱 Zero dependency: it uses [Petit-Dom](https://github.com/yelouafi/petit-dom) which is stable and embedded, so no bad suprise in the future
-- ♻️ Reactive: updating the state recalculate the Virtual Dom when needed
-- 🚀 fast: using virtual dom through a thin layer makes it close to bare-metal
-- 💧 Simple: that's [Vanilla](http://vanilla-js.com/), there isn't much to know, it's a raw class to extend; no magic ✨
-- 🪶 3 directives to enhance HTML: `:if` for conditions, `:for` for loops, `@myfuncname` to call an action
-- 🏡 Native: LEGO components are actual native web-components, you benefit from all the today's and tomorrow's possibilites ([slot](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots), [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM), …).
+LEGO is:
+
+- 👙 Minimalist: 117 lines of readable code in its core (non-optimized, uncompressed, no cheating) ;
+- 🌱 Low dependency: it uses [petit-dom](https://github.com/yelouafi/petit-dom) which is stable and embedded, and which itself has no dependency. So no bad surprise in the future ;
+- ♻️ Reactive: updating the state recalculate the [Virtual DOM](https://en.wikipedia.org/wiki/Virtual_DOM) when needed ;
+- 🚀 Fast: using Virtual DOM through a thin layer makes it close to bare-metal ;
+- 💧 Simple: that's [VanillaJS](http://vanilla-js.com/)! There isn't much to know, it's a raw class to extend. No magic ✨ ;
+- 🪶 3 directives to enhance HTML: `:if` for conditions, `:for` for loops, `@myfuncname` to call an action ; <!-- FIXME: Should we keep this line? Seems it has been removed in v2 doc. -->
```

Comment:
:warning: Should we keep this line? Seems it has been removed in v2 doc.

Reply:
I'd keep it